main.py

```
@@ -1,75 +1,194 @@
+            "page": page,
+            "limit": limit,
+            "total": 100
+        }
+    }
+
 
 # 내가 좋아요한 게시글 목록
-@app.get("/users/me/likes")
+@app.get("/users/me/likes", response_model=user.MyLikesResponse)
 async def get_user_likes(
         page: int = Query(default=1, ge=1, description="페이지 번호"),
```

Ruff (0.14.11)

Unused function argument: `page`

(ARG001)